src/BaseSchema.test.js

```diff
@@ -178,6 +178,16 @@ describe('BaseSchema', () => {
         })
       })
 
+      it('root-level required', () => {
+        expect(() => {
+          return S.object().required().valueOf()
+        }).toThrowError(
+          new S.FluentSchemaError(
+            "'required' has called on root-level schema, check your calls to require()"
```

Comment:
`require()` => `required()`

Reply:
It's duplicate of https://github.com/fastify/fluent-json-schema/pull/134#discussion_r644350063